client/src/templates/Challenges/redux/code-storage-epic.js

```diff
@@ -96,16 +96,17 @@ function saveCodeEpic(action$, state$) {
       }
     }),
     ofType(types.saveEditorContent),
-    switchMap(({ error }) =>
-      of(
+    switchMap(({ error }) => {
+      return of(
```

Comment:
Unless I'm missing something, this `return` isn't necessary.  The messages are clearer, though!

Reply:
Oops! forgot to remove that.  I'll fix that now